src/components/accessibility/Skiplink.jsx

```diff
@@ -16,9 +18,22 @@ export const Skiplink = ({
   ...props
 }) => {
   const ref = useRef(null);
+  const currentLocation =
+    typeof window !== 'undefined' ? window.location.pathname : '';
+  const currentSearch =
+    typeof window !== 'undefined' ? window.location.search : '';
+  const currentUrl = `${currentLocation}${currentSearch}`;
   return (
     <a
       href={anchorLink}
+      onClick={(e) => {
+        e.preventDefault();
+        navigate(currentUrl + anchorLink);
+        const elem = document.getElementById(anchorLink.replace('#', ''));
+        elem.setAttribute('tabindex', -1);
+        elem.focus({ preventScroll: true });
+        scrollTo(anchorLink);
```

Comment:
The -1 tab index will also remove elements form the tab order if they're focuable by default. I don't think we're doing that anywhere currently, but that could pitfall for a future developer.

We could always store and restore the existing tab index:

```suggestion
        const prevTabIndex = elem.tabIndex;
        elem.setAttribute('tabindex', -1);
        elem.focus({ preventScroll: true });
        elem.setAttribute('tabindex', prevTabIndex);
        scrollTo(anchorLink);
```

Reply:
We weren't previously skipping to anything that was tabbable but good call out on the future dev pitfall. Thanks for the GH suggestion.